app/models/admin.rb

```diff
@@ -0,0 +1,8 @@
+class Admin < ActiveRecord::Base
+  has_secure_password
+
+  validates :first_name, :last_name, :email, presence: true
+  validates :email, format: {
+              with: /\b[A-Z0-9._%a-z\-]+@(?:[A-Z0-9a-z\-]+\.)+[A-Za-z]{2,4}\z/
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.